pkg/certificate/analyzer.go

```diff
@@ -62,26 +62,29 @@ func (a *analyzer) start() {
 			// cap the list to an arbitrary amount (hopefully smaller than the 16 MB document size cap)
 			// anything approaching this limit will cause performance issues in software that depends on rita
 			// anything tuncated over this limit won't be visible as an IP connecting to an invalid cert
-			if len(datum.OrigIps) > 200003 {
-				datum.OrigIps = datum.OrigIps[:200003]
+			origIPs := datum.OrigIps.Items()
```

Comment:
Note that .Items() must be called in order to convert the map into an array. 

https://github.com/activecm/rita/commit/cd8dd38af38d30b06b0141d08b3370435ef2ba4e
https://github.com/activecm/rita/commit/180de847eb6ddb15e008891d9d690427ef307b45